packages/api/src/graphql/mutation.private.ts

```diff
@@ -717,6 +717,23 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
       }
     },
 
+    duplicateArticle: {
+      type: GraphQLNonNull(GraphQLArticle),
+      args: {
+        id: {type: GraphQLNonNull(GraphQLID)}
+      },
+      async resolve(root, {id}, {dbAdapter, loaders}) {
+        const article = await loaders.articles.load(id)
+
+        if(!article) throw new NotFound('article', id)
+
+        const articleRevision = Object.assign(article.published ?? article.draft, { slug: ''}, { publishedAt: undefined }, { updatedAt: undefined })
```

Comment:
you can do them in the same object. `{slug: '', publishedAt: undefined, updatedAt: undefined}`